prow/config/config.go

```diff
@@ -198,6 +198,8 @@ type Plank struct {
 	// JobURLPrefixConfig is the host and path prefix under which job details
 	// will be viewable. Use `org/repo`, `org` or `*`as key and an url as value
 	JobURLPrefixConfig map[string]string `json:"job_url_prefix_config,omitempty"`
+	// SkipRestartMissingPod will mark prowjob as error when associated pod is missing
+	SkipRestartMissingPod bool `json:"skip_restart_missing_pod,omitempty"`
```

Comment:
Is this supposed to be a feature gate or a config value?

AKA is this a feature gate that allows the sysadmin to control when it gets enabled? If so this should be a flag.

Or do we want sysadmins to permanently be able to choose whether or not to restart missing pods? If so then this makes sense as a config value.